docs/verifier/verification-library/zk-query-language.md

```diff
@@ -495,6 +894,32 @@ Please, check out our [<ins>Onchain verification tutorial to build a query</ins>
   }
 }
 ```
+</TabItem>
+</Tabs>
+
+
+
+## Proof of credential issuance
+
+Feature that enables requests to prove that credential exits in the ID Holder wallet without revealing any info.
```

Comment:
Shouldn't we mention here that this query somehow relates to the `$noop` operator?

Also, I'd consider a rephrasing of this description to:

By not including the `credentialSubject` in the query, a verifier requests proof of the presence of a credential in the user's wallet without requiring any condition to be met or revealing any specific detail about the credential.

Reply:
fixed to subject